WordPress/Jetpack/Classes/System/JetpackWindowManager.swift

```diff
@@ -5,25 +5,54 @@ class JetpackWindowManager: WindowManager {
     /// receives migration flow updates in order to dismiss it when needed.
     private var cancellable: AnyCancellable?
 
+    var shouldImportMigrationData: Bool {
+        return !AccountHelper.isLoggedIn && !UserPersistentStoreFactory.instance().isJPContentImportComplete
+    }
```

Comment:
**Non Blocking:** I think we don't want to import anything if the feature flag is disabled. I think the feature flag check should be removed from `shouldShowMigrationUI` and added here.